### DISCUSSION TITLE
Feature Request: Provide column order argument for `insert_into` python function

### DISCUSSION THREAD

**Discussion by darthf1** [duckdb/duckdb#12032] [Ideas]
https://duckdb.org/docs/sql/statements/insert#insert-column-order

The `INSERT INTO` statements allows the `BY POSITION` or `BY NAME` column order argument, however this argument cannot be passed into the python API; `BY POSITION` is the only and default operation.

```python
    def insert_into(self, table_name): # real signature unknown; restored from __doc__
        """
        insert_into(self: duckdb.duckdb.DuckDBPyRelation, table_name: str) -> None
        
        Inserts the relation object into an existing table named table_name
        """
        pass
```

**Comment by Tishj** [comment]
You can use `append`

https://github.com/duckdb/duckdb/pull/7300

**Reply by Tishj** [reply]
Ah actually, different issue, but somewhat related
Yea we should add that, feel free to make a PR 👍

**Comment by darthf1** [comment]
> Ah actually, different issue, but somewhat related Yea we should add that, feel free to make a PR 👍

I would love to, but unfortunately I'm not a C++ developer. Hopefully someone else with the right skill set can contribute :)
